.github/workflows/Python_tests.yml

```diff
@@ -0,0 +1,36 @@
+# TODO: Line 14, enable os: windows-latest
+# TODO: Line 15, enable python-version: 3.5
+# TODO: Line 36, enable pytest --doctest-modules
+
+name: Python_tests
+on: [push]
+jobs:
+  Python_tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      max-parallel: 15
+      matrix:
+        os: [macos-latest, ubuntu-latest]  # , windows-latest]
```

Comment:
Why Windows not get supported ? A block from github action ?

Reply:
There are three todos at the top of the file that we can work on after it lands.  This is one of them because the Windows doctests fail because they try to exersize some dead code that tries to the version of VSCode or similar which we now do elsewhere.